### DISCUSSION TITLE
Eclipse Theia v1.51.0

### DISCUSSION THREAD

**Discussion by jfaltermeier** [eclipse-theia/theia#13850] [Release Announcements] [answered]
Hi everyone 👋,

The Eclipse Theia v1.51.0 release is scheduled for later today!
We will inform you once the release starts and completes.

**Comment by tsmaeder** [comment]
We have a couple of PR's which we'd like to have in the release still open.

**Reply by rschnekenbu** [reply]
At least for vscode compat, we would like these PRs to be merged: 
- #13839 
- #13847
- #13849

I was also that #13782 was accepted and it could be merged on master before release.

**Reply by rschnekenbu** [reply]
#13839 & #13847 have been merged. The API bump is the only remaining task for VS Code compat. I am not aware of any other tasks to be merged before this release.

**Comment by tsmaeder** [comment]
- #13782 was accepted and it could be merged on master before release.

**Comment by tsmaeder** [comment]
I'm currently testing https://github.com/eclipse-theia/theia/pull/13309. Should have a go/no-go by noon.

**Reply by msujew** [reply]
We've decided to delay the feature in favor of more breaking changes to the underlying protocol before the official release. No need to review it before the release. Thanks for looking into it though 👍

**Comment by rschnekenbu** [comment]
All pending PRs on vscode API have been merged. Thanks @tsmaeder!

**Comment by tsmaeder** [comment]
@jfaltermeier as far as I'm concerned, all PR's we wanted in the release have been merged. Fire away!

**Comment by jfaltermeier** [comment]
The release will start now, I’ll post an update when it has completed.

**Comment by jfaltermeier** [comment]
The release has completed, thank you to everyone that participated and contributed!

Just as a fyi, there was one commit merged to the main branch while the release was created, that is therefore not included https://github.com/eclipse-theia/theia/commit/f996098ad43bc8dc940fc51d74429b2e5ab3a973
The tag for the release is created on the release branch, which reflects the actual contents of the release

**Comment by JonasHelming** [comment, marked as answer]
[News & Noteworthy](https://eclipsesource.com/blogs/2024/07/04/eclipse-theia-1-51-release-news-and-noteworthy/)
